### PR TITLE
[visq-unittest] Initialize visq-unittest module

### PR DESCRIPTION
**Pull request by jinevening**
This initializes visq unittest module.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Draft: https://github.com/Samsung/ONE/pull/10327

### CODE REVIEW

File: compiler/visq-unittest/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+###
+### Import visqlib module
+###
+get_target_property(VISQ_BIN_PATH visq BINARY_DIR)
+add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/visqlib
+                   COMMAND ${CMAKE_COMMAND} -E create_symlink
+                   ${VISQ_BIN_PATH}/visqlib ${CMAKE_CURRENT_BINARY_DIR}/visqlib)
+
+list(APPEND VISQ_TEST_DEPS ${CMAKE_CURRENT_BINARY_DIR}/visqlib)
```
Comment (jinevening) [inline compiler/visq-unittest/CMakeLists.txt:24]: This part is the difference between `visq-unittest` and the previous test codes in visq.

With this code, `visq-unittest` can access visqlib to test Python modules.

**Review by mhs4670go — APPROVED** [review]
LGTM

File: compiler/visq-unittest/test/testPalette.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (seanshpark) [inline compiler/visq-unittest/test/testPalette.py:1]: maybe 2023? or set other files to 2022?

Comment (jinevening) [reply compiler/visq-unittest/test/testPalette.py:1]: Files in this PR are not newly created but moved from `visq`. So the copyright year is different file by file :)

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**